lib/flatfile/csvActorExport.class.php

```diff
@@ -57,28 +57,27 @@ private function exportRelations($filename, $resource)
   {
     $rows = array();
 
-    foreach ($resource->getActorRelations() as $item)
+    foreach ($resource->getActorRelations() as $relation)
     {
-      $relatedEntity = $item->getOpposedObject($resource->id);
+      $relatedEntity = $relation->getOpposedObject($resource->id);
 
-      if (QubitTerm::ROOT_ID == $item->type->parentId)
+      // Determine appropriate relation type term
+      $relationType = $relation->type;
+      if ($relation->subjectId == $resource->id)
       {
-        $category = $item->type;
-      }
-      else
-      {
-        $category = $item->type->parent;
+        $converseRelation = $relationType->getConverseActorRelationTerm();
+        $relationType = (empty($converseRelation)) ? $relationType : $converseRelation;
       }
 
       $rows[] = array(
-        'sourceAuthorizedFormOfName' => $resource->authorizedFormOfName,
-        'targetAuthorizedFormOfName' => $relatedEntity->authorizedFormOfName,
-        'category'                   => (string)$category, // Return string representation for QubitTerm
-        'description'                => $item->description,
-        'date'                       => $item->date,
-        'startDate'                  => $item->startDate,
-        'endDate'                    => $item->endDate,
-        'culture'                    => $resource->culture
+        'objectAuthorizedFormOfName'  => $resource->authorizedFormOfName,
+        'subjectAuthorizedFormOfName' => $relatedEntity->authorizedFormOfName,
+        'relationType'                => (string)$relationType, // Return string representation for QubitTerm
```

Comment:
Does `(string)$relationType` return the correct i18n value if the culture is not 'en'?

Reply:
Hmm... good question! I'll test...

Reply:
Basically how CSV exports seem to work, based on how the I.O. export works, is that everything gets exported in the site's default language (usually English) so there's no cultural fallback when exporting something that doesn't exist in the default language (otherwise if you round-tripped the data you'd get translations showing up as the default language).

So seems like it's working correctly?

Reply:
Yeah, now that you mention it I recalling rumors that the CSV export doesn't work in other cultures... :(